plugins/eventMacro/eventMacro/Condition/VarDefined.pm

```diff
@@ -0,0 +1,38 @@
+package eventMacro::Condition::VarDefined;
+
+use strict;
+
+use base 'eventMacro::Condition';
+
+use eventMacro::Data;
+use eventMacro::Utilities qw(find_variable);
+
+sub _parse_syntax {
+	my ($self, $condition_code) = @_;
+
+	if ($condition_code =~ /^$general_variable_qr/) {
+		
+		if ($var =~ /^\./) {
```

Comment:
i got error here 
`An error occured while loading condition module eventMacro::Condition::VarDefined:Global symbol "$var" requires explicit package name at plugins/eventMacro/eventMacro/Condition/VarDefined.pm line 15.`

Reply:
oh, my bad, i forgot that part